common/djangoapps/student/models.py

```diff
@@ -837,7 +837,12 @@ def user_post_save_callback(sender, **kwargs):
 
     # Ensure the user has a profile when run via management command
     _called_by_management_command = getattr(user, '_called_by_management_command', None)
-    if _called_by_management_command and kwargs['created']:
+    log.info('Got post signal for  new profile for user: %s', user)
```

Comment:
If sandbox creationd works fine remove the logs.

Reply:
Removed the logs.